hugo-modules/core/utils/seo/private/get-tags.html

```diff
@@ -42,16 +40,22 @@
 {{ with $seo.locale }}
   {{ $c := dict "key" "locale" "value" .}}
   {{ $tags = $tags | append (partialCached "utils/seo/tag/og" $c $c) }}
+
+  {{ $attr := dict "rel" "alternate" "href" $.Permalink "hreflang" . }}
+  {{ $c2 := dict "name" "link" "attr" $attr }}
+  {{ $tags = $tags | append $c2 }}
 {{ end }}
 
+{{ warnf $.Permalink }}
```

Comment:
Was this just for debugging or do we need it for logging in production?

Reply:
Ah, yes, L49 should go

Reply:
And gone